update ds/wblt (#48)
OI-wiki/i18n: docs/ds/wblt.md

author: hsfzLZH1, cesonic

WBLT, abbreviation for weight balanced leafy tree, is an uncommon way of implementing a balanced tree. However, it has the advantage that its constant is relatively small and can be used as [mergeable heap](https://en.wikipedia.org/wiki/Mergeable_heap).

> Similar to WBT (weight-balanced trees), WBLT shows the propertity of leafy, that is, it has many nodes. But how many?

Reviewer: how many 🌚

Author: 还有property又打错了 🌚

Author: 💭 
> But how many can be counted as many? 
还是直接删除算了lol

> For $n$ numbers, unlike [treap](./treap) and others, WBLT will create $2n$ nodes. The weight of each node is the weight of its right child, and weight of the right child is greater than or equal to the left child.

Reviewer: create 2n nodes with the weight equal to that of its right child, which is greater than or equal to left child?

> Each insertion, similar to the heap, we swap one by one downwards sand pushup upwards. The operation is the same for deletion.

Reviewer: ```suggestion
Each insertion, similar to the heap, we swap one by one downwards and update with `pushup`. The operation is similar for deletion.
```

WBLT, abbreviation for weight balanced leafy tree, is an uncommon way of implementing a balanced tree. However, it has the advantage that its constant is relatively small and can be used as [mergeable heap](https://en.wikipedia.org/wiki/Mergeable_heap).

> Similar to WBT (weight-balanced trees), WBLT shows the property of leafy, that is, it has many nodes. But why we would call it many?

Reviewer: ```suggestion
Similar to WBT (weight-balanced trees), WBLT shows the property of leafy, that is, it has many nodes. But why we would we say that?
```

WBLT, abbreviation for weight balanced leafy tree, is an uncommon way of implementing a balanced tree. However, it has the advantage that its constant is relatively small and can be used as [mergeable heap](https://en.wikipedia.org/wiki/Mergeable_heap).

> Similar to WBT (weight-balanced trees), WBLT shows the property of leafy, that is, it has many nodes. But how many can be counted as many?

Reviewer: 这里“怎么多”感觉还是说“为什么我们说他节点多”
